docker/assets/grafana/dashboards/analytics_dashboard.json

```diff
@@ -43,6 +43,7 @@
         "type": "influxdb",
         "uid": "PE5723DBC504634E6"
       },
+      "description": "",
```

Comment:
just curious...this necessary?

Reply:
Probably not. Grafana does some weirdness to the JSON depending on where you click in the GUI. I can remove it.

```suggestion
```